platform/ios/display_server_ios.h

```diff
@@ -79,6 +79,9 @@ class DisplayServerIOS : public DisplayServer {
 
 	int virtual_keyboard_height = 0;
 
+	DisplayServerIOS::IOSUIGestureState ui_gesture_state = static_cast<DisplayServerIOS::IOSUIGestureState>((int)GLOBAL_GET("display/window/ios/ui_gesture_state"));
+	bool status_bar_hidden = GLOBAL_GET("display/window/ios/hide_status_bar");
```

Comment:
You can't declare this in the header with `GLOBAL_GET`

Reply:
What would be the preferred method? The default state on launch needs to be the same as the project setting or it could cause buggy behavior.

Reply:
In the constructor

Reply:
Like is done for tts

Reply:
On it